docs/src/modules/components/MaterialStartingLinksCollection.js

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import Grid from '@mui/material/Unstable_Grid2';
+import InstallDesktopRoundedIcon from '@mui/icons-material/InstallDesktopRounded';
+import WebRoundedIcon from '@mui/icons-material/WebRounded';
+import DrawRoundedIcon from '@mui/icons-material/DrawRounded';
+import PlayCircleFilledWhiteRoundedIcon from '@mui/icons-material/PlayCircleFilledWhiteRounded';
+import DesignServicesRoundedIcon from '@mui/icons-material/DesignServicesRounded';
+import InfoCard from 'docs/src/components/action/InfoCard';
+
+const content = [
+  {
+    title: 'Installation',
+    description: 'Add Material UI to your project with a few commands.',
+    link: '/material-ui/getting-started/installation/',
+    icon: <InstallDesktopRoundedIcon color="primary" />,
+  },
+  {
+    title: 'Usage',
+    description: 'Learn the basics about Material UI components.',
+    link: '/material-ui/getting-started/usage/',
+    icon: <DrawRoundedIcon color="primary" />,
+  },
+  {
+    title: 'Example projects',
+    description: 'A collection of boilerplates to jumpstart your next project.',
+    link: '/material-ui/getting-started/example-projects/',
+    icon: <PlayCircleFilledWhiteRoundedIcon color="primary" />,
+  },
+  {
+    title: 'Customizing components',
+    description: 'Learn about the available customization methods.',
+    link: '/material-ui/customization/how-to-customize/',
+    icon: <DesignServicesRoundedIcon color="primary" />,
+  },
+  {
+    title: 'Templates',
```

Comment:
I wonder if we should link this to MUI Store templates, or change the title to 'Free templates' and keep the link to the doc's templates. What do you think?

Reply:
The Templates page on the Getting Started section has a link to the Store (calling it "Premium templates"), so I wonder if we really need to discriminate it as free?

Reply:
I think it works too, no strong opinions about it.